documentation/source/building.rst

```diff
@@ -1,3 +1,5 @@
+.. _building:
```

Comment:
This shouldn't be needed, you can already reference this as ``:doc:`building` ``

Reply:
This is for resilience against file renamings likely to happen in the future.